Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
 
-  if (!strcmp((const char*)(s_cmd_bfr + 1), "TStatus"))
-  {
-    return SendReply("T0");
-  }
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "Attached"))
+    return SendReply("1");
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "C"))
+    return SendReply("QC1");
```

Comment:
The [documentation](https://sourceware.org/gdb/onlinedocs/gdb/General-Query-Packets.html#index-qC-packet) says that the syntax is <code>QC <var>thread-id</var></code>, but it also says near the top:

> We include spaces in some of the templates for clarity; these are not part of the packet’s syntax. No GDB packet uses spaces to separate its components.

I assume that this is correct, but I'd like to make sure.

Reply:
This is correct, you actually see this nomenclature everywhere.

Btw, you can actually log from GDB's side what it sends and receive by using `set debug remote on` before connecting which can be done with `target remote localhost:port` where `port` is the one you set in the INI. (you might want to do `set architecture powerpc:750` and `set endian big` before though)